Nodejs/Product/Nodejs/GeneratedHelpAboutVersion.tt

```diff
@@ -0,0 +1,23 @@
+﻿<#@ template debug="false" hostspecific="true" language="C#" #>
+<#@ parameter type="System.String" name="BuildVersion" #>
+<#@ output extension=".cs" #>
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.NodejsTools
+{
+    // This attribute is used to register the information needed to show this package
+    // in the Help/About dialog of Visual Studio.
+    [InstalledProductRegistration("#110", "#112", "<#= BuildVersion ?? "dev machine" #>", IconResourceID = 400)]
```

Comment:
Is this just a free-form string? Is it not better as "42.42.42.42" or whatever we use for the version on dev machines?